### ISSUE TITLE
Why do you perform L2 normalization on attributions ？

### ISSUE THREAD

**Issue by q642936448** [pytorch/captum#859]
Hello，I notice that you always perform L2 normalization on attributions. But why? 
for example, in `IMDB_TorchText_Interpret.ipynb`:
```
def add_attributions_to_visualizer(attributions, text, pred, pred_ind, label, delta, vis_data_records):
    attributions = attributions.sum(dim=2).squeeze(0)  # WHY???
    attributions = attributions / torch.norm(attributions)
    attributions = attributions.cpu().detach().numpy()

    # storing couple samples in an array for visualization purposes
    vis_data_records.append(visualization.VisualizationDataRecord(
                            attributions,
                            pred,
                            Label.vocab.itos[pred_ind],
                            Label.vocab.itos[label],
                            Label.vocab.itos[1],
                            attributions.sum(),
                            text,
                            delta))
```
Maybe the reason is that there are postive and negative attributions, But what is the advantage of using L2 normalization?

Can I use L1 normalization instead of L2 ?

Thanks in advance :)

**Comment by NarineK** [comment]
@q642936448, L2 is just an example, the difference between L2 and L1 is the scale. L2 is used to make them smaller scale, if you want larger scale you can use L1. There is no rule which one to use and it has no connection to attribution score being positive or negative. If we scale something, then we know the range of that scale. Hope this is clear.

**Comment by NarineK** [comment]
@q642936448, if this answers to your question do you mind, please, closing the issue ?

**Comment by q642936448** [comment]
Tx😁Your answer helps a lot!
